src/App/src/Handler/HomePageHandlerFactory.php

```diff
@@ -20,6 +20,8 @@ public function __invoke(ContainerInterface $container): RequestHandlerInterface
             ? $container->get(TemplateRendererInterface::class)
             : null;
 
+        // @codingStandardsIgnoreStart
```

Comment:
What was the reported issue here? :thinking:

Reply:
```FILE: src/App/src/Handler/HomePageHandlerFactory.php
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------
 23 | ERROR | [x] Class name referenced via call of function get_class().
    |       |     (SlevomatCodingStandard.Classes.ModernClassNameReference.ClassNameReferencedViaFunctionCall)
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------

Time: 81ms; Memory: 10MB
```

Reply:
`@codingStandardsIgnoreStart` is deprecated, `phpcs:disable` should be used. For a single line `phpcs:ignore` can be used, which causes a little less noise.
https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-parts-of-a-file